docs/application/native/guides/index.md

```diff
@@ -8,275 +8,495 @@ The native API specification consists of multiple API modules that represent the
-| | [External Output Manager (since 2.4)](../api/mobile/latest/group__CAPI__UI__EOM__MODULE.html) | Provides functions for external outputs. | The UI API module contains submodule APIs and open source libraries for 2D and 3D graphics and text. <br>Guides:<br>  - [Application Management](app-management/index.md)<br>  - [Device Settings and Systems](device/overview.md)<br>  - [Graphics](graphics/overview.md)<br>  - [Notifications and Content Sharing](notification/overview.md)<br>  - [User Interface](ui/index.md)|
-| | [Fontconfig](https://developer.tizen.org/dev-guide/latest/../api/wearable/latest/group__OPENSRC__FONTCONFIG__FRAMEWORK.html) and [Freetype](https://developer.tizen.org/dev-guide/latest/../api/wearable/latest/group__OPENSRC__FREETYPE__FRAMEWORK.html) | Provides a text rendering library and font-handling library to let applications find a font or closely matching font. | The UI API module contains submodule APIs and open source libraries for 2D and 3D graphics and text. <br>Guides:<br>  - [Application Management](app-management/index.md)<br>  - [Device Settings and Systems](device/overview.md)<br>  - [Graphics](graphics/overview.md)<br>  - [Notifications and Content Sharing](notification/overview.md)<br>  - [User Interface](ui/index.md)|
-| | [HarfBuzz (since 2.4)](../api/mobile/latest/group__OPENSRC__HARFBUZZ__FRAMEWORK.html) | Provides functions for text shaping.  | The UI API module contains submodule APIs and open source libraries for 2D and 3D graphics and text. <br>Guides:<br>  - [Application Management](app-management/index.md)<br>  - [Device Settings and Systems](device/overview.md)<br>  - [Graphics](graphics/overview.md)<br>  - [Notifications and Content Sharing](notification/overview.md)<br>  - [User Interface](ui/index.md)|
-| | [Minicontrol (since 2.4)](../api/mobile/latest/group__MINICONTROL__LIBRARY.html) | Provides functions for creating and displaying an EFL socket window. | The UI API module contains submodule APIs and open source libraries for 2D and 3D graphics and text. <br>Guides:<br>  - [Application Management](app-management/index.md)<br>  - [Device Settings and Systems](device/overview.md)<br>  - [Graphics](graphics/overview.md)<br>  - [Notifications and Content Sharing](notification/overview.md)<br>  - [User Interface](ui/index.md)|
-| | [OpenGL ES](../api/mobile/latest/group__OPENSRC__OPENGLES__FRAMEWORK.html) **(since 4.0)** | Provides a library for rendering 3D and 2D graphics in embedded systems. | The UI API module contains submodule APIs and open source libraries for 2D and 3D graphics and text. <br>Guides:<br>  - [Application Management](app-management/index.md)<br>  - [Device Settings and Systems](device/overview.md)<br>  - [Graphics](graphics/overview.md)<br>  - [Notifications and Content Sharing](notification/overview.md)<br>  - [User Interf
```

Comment:
I compared the list with the latest 5.0 native API on TD, 
and found some sub modules are not included in this list.

Would you please check below items,
which are in the api tree but not in this list?

**Mobile**

- Network
   - Intelligent Network Monitoring
   - SoftAP

- Security
   - Device Certificate Manager

- System
   - Dlog (different order with TD)

- UIX
   - Multi assistant
   - Voice Contorl manager

**Wearable**
- Application Framework
   - RPC Port
   - Watchface complication
- Messaging
   - Copy&paste error
- Network
   - Intelligent Network Monitoring
   - SoftAP
- Security
   - Device Certificate Manager
- System
   - Dlog (different order with TD)
- UI
   - Minicontrol
- UIX
   - Multi assistant
   - Voice Control manager

Reply:
The description of Dlog is correct and more detailed. I think it doesn't need to same as the overview paragraph of API reference.

This change is based on the master branch contents, but I will update additional modules since 4.0 and 5.0.